### PR TITLE
feat(tests): add einval tests

### PR DESCRIPTION
**Pull request by saidsay-so**

### CODE REVIEW

File: rust/src/tests/ftruncate.rs [hunk 1]
```diff
@@ -15,8 +15,6 @@ use crate::{
     utils::chmod,
 };
 
-// tests/ftruncate/00.t
```
Comment (asomers) [inline rust/src/tests/ftruncate.rs:18]: Why remove this comment?

**Review by asomers — COMMENTED** [review]
Looks good, but what about ftruncate/12.t and truncate/12.t?  Also, don't forget the ENOTEMPTY part of rmdir/12.t.

**Comment by asomers** [comment]
Oh, I just realized that the truncate/12.t and ftruncate/12.t test cases were in a different PR.

**Comment by musikid** [comment]
> Looks good, but what about ftruncate/12.t and truncate/12.t? Also, don't forget the ENOTEMPTY part of rmdir/12.t.

I already added the ENOTEMPTY part in the eexist branch (not yet merged).

File: rust/src/tests/ftruncate.rs (hunk 1 above)
Comment (musikid) [reply rust/src/tests/ftruncate.rs:18]: It's just a mistake.

**Review by asomers — APPROVED** [review]

**Pull request merged**